lambdas/db-migration/src/migrations/20201014105045_create_executions_table.ts

```diff
@@ -9,6 +9,10 @@ export const up = async (knex: Knex): Promise<void> =>
       .text('arn')
       .comment('Execution ARN')
       .notNullable();
+    table
+      .text('name')
```

Comment:
We could store the `name` in a text field, but we could also use the explicit UUID field type, since that's what the data already is. Honestly I'm not totally sure of what the benefits are of using that field type though

Reply:
From a DS colleague

> if it is the UUID type, it will be stored as a binary representation, so will be smaller on disk (and indexes will be smaller too) but performance etc should be the the same

So seems like it might be worth using the `uuid` field type?